src/components/IconGithub.jsx

```diff
@@ -1,16 +1,44 @@
+import React, { useState, useEffect } from 'react';
+
 export default function IconGithub() {
+  const [stars, setStars] = useState(0);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(`https://api.github.com/repos/markmead/hyperui`);
+        const data = await response.json();
+        const starCount = data.stargazers_count;
+        setStars(starCount);
```

Comment:
Can we do some destructuring to clean it up?

Reply:
please review the recent commit.